rc_dev

```diff
@@ -4,3 +4,7 @@ export API_URL=//mf-chsdi3.dev.bgdi.ch
 export APACHE_BASE_PATH=
 export BROWSERSTACK_TARGETURL=https://mf-geoadmin3.dev.bgdi.ch
 export VARNISH_HOSTS=()
+export DEFAULT_TOPIC_ID=ech
+export TRANSLATION_FALLBACK_CODE=de
+export DEFAULT_EXTENT='[485869.5728, 76443.1884, 837076.5648, 299941.7864]'
+export DEFAULT_RESOLUTION=500.0
```

Comment:
Please put the default values directly in the Makefile, where it could/can be overwritten withe environment variabls; similar to https://github.com/geoadmin/mf-geoadmin3/blob/master/Makefile#L18. Otherwise, we can't build without these variables being defined. Also, like this, you don't need to put default values in any rc_* file.

You can overwrite those values using any rc_* file then.